lib/usbd/backend/usbd_dwc_otg.c

```diff
@@ -109,12 +109,25 @@ static inline uint16_t get_fifo_depth(usbd_device *dev)
  */
 static void flush_fifo(usbd_device *dev)
 {
+	// Set Global IN OUT ep NACK
```

Comment:
i also would like to point out https://github.com/insane-adding-machines/unicore-mx/blob/master/HACKING
it is a useful resource for stuff like coding style we accept.